src/test/java/com/example/enterprise/MoneyManagerApplicationTests.java

```diff
@@ -96,7 +96,7 @@ void addAndSaveANewExpense(){
 //    Get Income
     @Test
     void checkReturnIncomeList(){
-        incomeService.listAll();
+        incomeService.listAllIncomes();
```

Comment:
Changed the method name from listAll() to listAllIncomes() because listAll doesn't specify what to list. An outsider would not understand what it is trying to list. listAllIncomes() name is much more meaningful and provides more clarity to the code logic.